### PR TITLE
makepkg: refresh

### PR DESCRIPTION
**Pull request by CleanMachine1**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** 6.0.1

### CODE REVIEW

**Review by 258204 — APPROVED** [review]
Can't find anything I would change.

File: pages/linux/makepkg.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
 # makepkg
 
-> Creates a package installable with the `pacman` package manager.
-> Runs the commands from a `PKGBUILD` file to build the package.
+> Creates a package which can be used with `pacman`.
+> Uses the `PKGBUILD` file in the current working directory.
```
Comment (mfrw) [inline pages/linux/makepkg.md:4]: Should we mention `current working dir` here ?
It does take an option `-p` where we can supply any `PKGBUILD` file.
How about:
```suggestion
> Uses the `PKGBUILD` file in the current working directory by default.
```

**Review by mfrw — COMMENTED** [review]
Should we specify that `PGKBUILD` file is the default action ?

File: pages/linux/makepkg.md [hunk 2]
```diff
@@ -1,22 +1,22 @@
 # makepkg
 
-> Creates a package installable with the `pacman` package manager.
-> Runs the commands from a `PKGBUILD` file to build the package.
+> Creates a package which can be used with `pacman`.
+> Uses the `PKGBUILD` file in the current working directory by default.
```
Comment (navarroaxel) [inline pages/linux/makepkg.md:4]: I think this sounds better
```suggestion
> Create a package which can be used with `pacman`.
> Use the `PKGBUILD` file in the current working directory by default.
```

Comment (CleanMachine1) [reply pages/linux/makepkg.md:4]: I agree with the first line however the second line change seems out of place.

Comment (mfrw) [reply pages/linux/makepkg.md:4]: Yep, reading it on a new day, it does seem odd :(
We can maybe omit the `by default`, given we start with `Use`.

Comment (sbrl) [reply pages/linux/makepkg.md:4]: I also agree with @CleanMachine1.

Comment (mfrw) [reply pages/linux/makepkg.md:4]: Maybe this sounds better ? (thoughts)
```suggestion
> Creates a package which can be used with `pacman`.
> Uses the `PKGBUILD` file in the current working directory to build.
```

Comment (CleanMachine1) [reply pages/linux/makepkg.md:4]: I think axel's edited suggestion is sufficient

File: pages/linux/makepkg.md [hunk 3]
```diff
@@ -1,22 +1,22 @@
 # makepkg
 
-> Creates a package installable with the `pacman` package manager.
-> Runs the commands from a `PKGBUILD` file to build the package.
+> Create a package which can be used with `pacman`.
+> Uses the `PKGBUILD` file in the current working directory by default.
```
Comment (mfrw) [inline pages/linux/makepkg.md:4]: I think this reads fine to me. Thanks @CleanMachine1

**Review by mfrw — APPROVED** [review]
LGTM

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**